extensions/amp-story/1.0/animation.js

```diff
@@ -617,7 +615,7 @@ export class AnimationManager {
         )
         .concat(
           Array.prototype.map.call(
-            this.root_.querySelectorAll('amp-story-animation'),
+            this.page_.querySelectorAll('amp-story-animation'),
```

Comment:
The `:scope > amp-story-animation`, so let's do `childElementsByTag` from `dom.js`?